content/applications/finance/fiscal_localizations/united_states.rst

```diff
@@ -0,0 +1,746 @@
+    on payslips.
+  - :guilabel:`Bank Account Number`: the bank account associated with the NACHA payment file.
+
+- :guilabel:`HR Settings` tab:
+
+  - :guilabel:`Federal Tax Filing Status`: the tax status an employee uses for Payroll tax
+    calculations, which can be different from their state status.
+  - :guilabel:`State Tax Filing Status`: the tax status an employee uses for their state portion of
+    the Payroll tax calculation.
+  - :guilabel:`W-2 Form`: a US tax form indicating the summary of wages, taxes, and benefits paid to
+    an employee during a tax period (typically one year).
+  - :guilabel:`W-4 Form`: an IRS form that helps outline the amount of federal taxes to withhold for
+    an employee, which is paid to the IRS by the company.
+
+.. _l10n_us/payroll-employee-contracts:
+
+Employee contracts
+~~~~~~~~~~~~~~~~~~
+
+Additionally, there is information that is found in an employee contract that also affects the
+*Payroll* application.
+
+Navigate to the :menuselection:`Employees app --> Employees --> Contracts` and select a contract
+record to view the sections of a contract that directly affect *Payroll*:
+
+- :guilabel:`General Information`:
+
+  - :guilabel:`Salary Structure Type: United States: Employee`: defines when the employee is paid,
+    their working schedule, and the work entry type.
+  - :guilabel:`Work Entry source`: determines how work entries are calculated.
+
+- :guilabel:`Salary Information` tab:
+
+  - :guilabel:`SSN No`: the last four digits of the employee's Social Security Number (SSN) appears
+    on payslips.
+  - :guilabel:`Wage type`: determines how the employee is paid, wether a Fixed wage (salary) or
+    Hourly wage.
+  - :guilabel:`Schedule Pay`: defines how often the employee is paid, either :guilabel:`Annually`,
+    :guilabel:`Semi-annually`, :guilabel:`Quarterly`, :guilabel:`Bi-monthly`, :guilabel:`Monthly`,
+    :guilabel:`Semi-monthly`, :guilabel:`Bi-weekly`, :guilabel:`Weekly`, or :guilabel:`Daily`. In
+    the US, Semi-monthly (24 payments a year) or bi-weekly (26 payments a year) are the most common.
+  - :guilabel:`Wage, Yearly, and Monthly cost`: used to show the total cost of an employee. It is
+    recommended to populate the :guilabel:`Yearly` wage first, as it auto-populates the other
+    fields.
+  - :guilabel:`Pre-tax benefits`: populate this section according to the employee's selections.
+    Pre-tax benefits decrease the gross wage, which lowers the base amount that is taxed. These are
+    displayed at the beginning of the payslip.
+  - :guilabel:`Post-tax benefits`: these benefit are deductions made *after* taxes are calculated.
+    These appear towards the end of the payslip before the net amount is displayed.
+
+.. seealso::
+   :doc:`Employees documentation <../../hr/employees/new_employee>`
+
+.. _l10n_us/adp:
+
+Export work entries to ADP
+--------------------------
+
+Requirements
```

Comment:
For future improvements:
Think it's worth it to further explain each code to differentiate them more for the reader (is it *one* ADP code or is it *three* or more than that?), as far as what each code is/means and where to find them in ADP. A screenshot or two would be useful here too, given how important these steps.